linera-execution/tests/fee_consumption.rs

```diff
@@ -146,9 +146,10 @@ async fn test_fee_consumption(
         operation_byte: Amount::from_tokens(23),
         message: Amount::from_tokens(29),
         message_byte: Amount::from_tokens(31),
-        maximum_executed_block_size: 37,
-        maximum_bytes_read_per_block: 41,
-        maximum_bytes_written_per_block: 43,
+        maximum_fuel_per_block: 37,
```

Comment:
Is this just the primes? Curious if there is a reason or if it's just for aesthetics.

Reply:
I think so; I guess the numbers don't really matter for this test. I just continued the pattern. :shrug:

Reply:
Ah, of course the fuel limit _does_ matter. Finding a large prime number to make the test pass…

Reply:
It's good practice to use primes in tests like this since issues often show up down the pipeline presenting with numbers that are a product of the inputs, and if every input number is a unique prime then factoring a small integer is sufficient to tell you the culprit :)

Reply:
Then I guess multiple large primes aren't a good idea. :laughing:

Reply:
The computational definition of ‘large prime’ is hopefully larger than what you were thinking of adding :joy: